src/platforms/common/swo.h

```diff
@@ -23,8 +23,6 @@
 #ifndef PLATFORMS_COMMON_SWO_H
 #define PLATFORMS_COMMON_SWO_H
 
-#include <libopencm3/usb/usbd.h>
```

Comment:
As much as it means two `#ifndef NO_LIBOPENCM3` blocks, please wrap this line in na separate one so we can keep all includes at the top of the file.

Reply:
Alright, done